pkg/abi/nvgpu/ctrl.go

```diff
@@ -290,8 +291,9 @@ type RmapiParamNvU32List struct {
 
 // From src/common/sdk/nvidia/inc/ctrl/ctrl0080/ctrl0080gr.h:
 const (
-	NV0080_CTRL_CMD_GR_GET_CAPS = 0x801102
-	NV0080_CTRL_CMD_GR_GET_INFO = 0x801104
+	NV0080_CTRL_CMD_GR_GET_CAPS    = 0x801102
+	NV0080_CTRL_CMD_GR_GET_CAPS_V2 = 0x801109
```

Comment:
nit: NV0080_CTRL_CMD_GR_GET_CAPS_V2 should come after NV0080_CTRL_CMD_GR_GET_INFO since its value to greater. We try to keep these const blocks sorted.